README.md

```
@@ -1,17 +1,30 @@
+- [Example execution commands](#example-execution-commands)
+- [Output](#output)
+- [Code Overview](#code-overview)
+- [Development](#development)
+  - [Setting Up Development Environment](#setting-up-development-environment)
+  - [Required Commands Before Committing](#required-commands-before-committing)
+  - [CI Requirements](#ci-requirements)
+- [License](#license)
+
+### Prerequisites {#prerequisites}
 
 1. https://www.rust-lang.org/tools/install
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)